docs/getting-started.md

```diff
@@ -56,7 +56,7 @@ scalar.cassy.server.ssh_private_key_path=/path/to/.ssh/id_rsa
 # Path to the bin directory of Cassy in each Cassandra nodes (Cassy assumes all Cassandra nodes install Cassy in the same directory)
 scalar.cassy.server.slave_command_path=/path/to/cassy/build/install/cassy/bin
 
-# URI of a blob store to manage backup files 
+# URI of a blob store to manage backup files.(If the storage device is Azura, specify the name of the container)  
```

Comment:
```suggestion
# URI of a blob store to manage backup files.
```
I asked them to fix this to have the URI instead of container name. so let's leave it as it is.